### DISCUSSION TITLE
Bug: writes the answer from the Old Document even after new PDF is uploaded in the same chatbox.

### DISCUSSION THREAD

**Discussion by Urvesh71** [open-webui/open-webui#5855] [General]
# Bug Report

In Open Web UI front end, when I uploaded one document (PDF) and asked the question, I got an answer from the uploaded pdf's context. But in the same Chat box, I uploaded another different PDF document and I ask the question related the new PDF, I got an answer still from the Old PDF document. 

Therefore, I assume that when user replaces the old pdf with the new one, the Vector database, which contains the embeddings vector, does not get deleted or updated with the embeddings of the newly uploaded pdf (or document). 

Does anybody also find this problem? or if there is already a solution for this, please let me know. 

## Installation Method

I executed the Open Web UI using docker-compose.yml file. I included two services, Ollama and Open Web UI and ran the Open Web UI within the Docker container. 

## Environment

- **Open WebUI Version:** 
- **Ollama (if applicable):** 0.3.12

- **Operating System:** Windows 10
- **Browser (if applicable):** Microsoft Edge

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

User should always get the answer from the new (latest) uploaded pdf (document).

## Actual Behavior:

It still writes the answer of the question from the old uploaded PDF even after we upload the new pdf in the chat box.

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by Ducky6944** [comment]
Did you ever sort this out? I'm having the same issue but with a different "document." 

```
#https://www.youtube.com/watch?v=3sTsMVAMF1g
summarize this document. 
```

Gives expected result. 

```
#https://www.youtube.com/watch?v=yCnkkljii_k
summarize this document. 
```
Gives a rewrite based on the original "document." 

If you click on the document it does show the correct data to reference, but the model seems to be referencing the original data. The only way to resolve is to start a new chat.
